apps/meteor/app/lib/server/methods/saveSettings.js

```diff
@@ -41,6 +52,9 @@ Meteor.methods({
 					case 'multiSelect':
 						check(value, Array);
 						break;
+					case 'code':
+						parseToJSON(value);
```

Comment:
Not all `code` settings are JSON (eg some of them are CSS or HTML). We should check settings' `code` property to assure they should be a valid JSON.